Watcher/v1/summit.yaml

```diff
@@ -67,3 +67,54 @@ rules:
   - name: ScriptQueue:1
   - name: ScriptQueue:2
   - name: WeatherStation:1
+- classname: DewPointDepression
+  configs:
+  - name: AT
+    dew_point_sensors:
+    - sal_index: 201  # see https://confluence.lsstcorp.org/display/LTS/ESS+CSC+Index+Registry
+      topics:
+      - topic_name: hx85ba
+        sensor_names:
+        - AuxTel-ESS01
+    temperature_sensors:
+    - sal_index: 1
+      topics:
+      - topic_name: temperature
+        sensor_names:
+        - sensor_name: AuxTel-ESS02
+          indices: [1, 2, 3]
+    warning_level: 3
+    critical_level: 2
+    hysteresis: 0.2
+    poll_interval: 10
```

Comment:
As explained on Jira, the sensors output new data at least every 2 seconds. I guess a poll interval of 10 seconds is fast enough since temperatures don't tend to change drastically in 10 seconds.

Reply:
Thanks. I'll increase the poll rate. I'd rather use all the data.